apps/web/src/components/Renderers/RichTextRenderer/RichTextRender.tsx

```diff
@@ -0,0 +1,63 @@
+import type { Options } from '@contentful/rich-text-react-renderer'
+import { documentToReactComponents } from '@contentful/rich-text-react-renderer'
+import type { Document } from '@contentful/rich-text-types'
+import { BLOCKS, INLINES, MARKS } from '@contentful/rich-text-types'
+import Link from 'next/link'
+import type { FC, ReactNode } from 'react'
+import React from 'react'
+
+import { List, ListItem } from '@/components/molecules/List/List'
+
+function Bold({ children }: { children: ReactNode }) {
```

Comment:
this can be a one liner using more modern es6 syntax

`const Bold = ({ children }: { children: ReactNode }) => <span className="font-bold">{children}</span>`

In fact I would also create a shared type for this and `Text` and make it a one liner too